### PR TITLE
Pass Frame props to the iframe element

### PR DESCRIPTION
**Pull request by daviferreira**

### CODE REVIEW

**Comment by ryanseddon** [comment]
Even better let's just pass along this.props to the underlying iframe e.g.

```js
React.createElement('iframe', this.props);
```

This get's back to the equivalent transferProps.

**Comment by daviferreira** [comment]
:+1: 

That would have been my first approach, but I thought it would be too much since I only needed the className.

Want me to open another PR or you do it directly? :)

Thanks for this component btw, really great work

**Comment by ryanseddon** [comment]
In this PR is fine, just squash the commits or ammend this one.

File: test/Frame_spec.js [hunk 1]
```diff
@@ -30,6 +30,12 @@ describe("Frame test",function(){
     expect(frame.getDOMNode().style.border).toContain('0');
   });
 
+  it("should create an empty iFrame and apply given className", function () {
```
Comment (ryanseddon) [inline test/Frame_spec.js]: Make make this more generic `should pass along all props to underlying iFrame` or something along those lines.

Then have a few expectations with different props.

**Comment by ryanseddon** [comment]
Sweet thanks for the PR!

**Pull request merged**